plugin/click.py

```diff
@@ -0,0 +1,19 @@
+import sublime
```

Comment:
Maybe this code could live in `tooling.py`? Not sure such a tiny command requires its own file.